taigit/src/libraries/GitHub/src/avgComments.ts

```diff
@@ -1,80 +1,89 @@
-import axios from 'axios';
+import axios from "axios"
```

Comment:
Uh... what happened to this file? It looks like it was overwritten.

Reply:
This should fix the issue David has pointed out. It doesn't compile because avgComments is missing it's functions.

Reply:
Fixed it.